ansible/deploy-orchestration.yml

```diff
@@ -0,0 +1,33 @@
+---
+- include: ansible-version.yml
+
+- hosts: orchestration
+  gather_facts: true
+  roles:
+    - docker_py
```

Comment:
The single use of `orchestration` group I see is to install docker and docker is useless for `db-1` hosts. What's the point of that group?